src/gui/attributetable/qgsfeaturelistview.cpp

```diff
@@ -183,6 +183,18 @@ void QgsFeatureListView::editSelectionChanged( const QItemSelection &deselected,
     viewport()->update( visualRegionForSelection( localDeselected ) | visualRegionForSelection( localSelected ) );
   }
 
+  mLastEditSelectionFid = QgsFeatureId();
```

Comment:
can you rebase? there's a lot of unrelated commits here

Reply:
done !